ArduCopter/mode.h

```diff
@@ -1138,6 +1138,8 @@ class ModeGuided : public Mode {
         AllowWeatherVaning = (1U << 7)
     };
 
+    bool option_is_enabled(Option option) const;
```

Comment:
let's put a little comment above this one.  sorry, it's perhaps pretty obvious what it does but..

Reply:
Done